variables.tf

```diff
@@ -15,6 +15,12 @@ variable "api_servers_external" {
   default     = []
 }
 
+variable "api_servers_ips" {
```

Comment:
I would consider calling this variable `api_servers_cert_ips` if this variable is used *only* for adding the IPs to the certificates. Not sure this makes sense though, so will leave it up to you.